<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmPasien</class>
 <widget class="QMainWindow" name="FrmPasien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Pasien</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PASIEN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>501</width>
      <height>666</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="idOwnerLabel">
         <property name="text">
          <string>KTP</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Txt_KTP"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nama</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Txt_Nama"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="alamatLabel">
         <property name="text">
          <string>Alamat</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="Txt_Alamat"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="tempatlahirLabel">
         <property name="text">
          <string>Tempat lahir</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Txt_TempatLahir"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tanggal Lahir&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCalendarWidget" name="Cal_TanggalLahir"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelkelamin">
         <property name="text">
          <string>Kelamin</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="Cmb_Kelamin">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>pria</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>wanita</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labeltelepon">
         <property name="text">
          <string>Telepon</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="Txt_Phone"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="Chk_Kawin">
           <property name="text">
            <string>Kawin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Chk_Kawin_2">
           <property name="text">
            <string>Tidak Kawin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="PB_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_del">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
